### PR TITLE
add redirect for lexicon

### PR DESCRIPTION
**Pull request by laurieboyes**
@matthew-andrews 

Also: https://github.com/Financial-Times/next-opt-in-out/pull/161

### CODE REVIEW

File: server/controllers/lexicon.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+module.exports = (req, res) => {
+	res.redirect(301, `http://lexicon.ft.com${req.url.substring('/lexicon'.length)}`);
```
Comment (matthew-andrews) [inline server/controllers/lexicon.js]: if you add `ft_site=falcon` to the query param you'll get a message saying this page isn't available on Next yet :)

Comment (laurieboyes) [inline server/controllers/lexicon.js]: Whoops! Yep

Comment (matthew-andrews) [inline server/controllers/lexicon.js]: no whoops — it'll work fine without this :)

**Comment by matthew-andrews** [comment]
very lovely

**Pull request merged**